c/driver_manager/adbc_driver_manager.cc

```diff
@@ -720,7 +720,7 @@ struct ManagedLibrary {
         search_paths.emplace_back(SearchPathSource::kAdditional, path);
       }
 
-#if ADBC_CONDA_BUILD
+#if defined(ADBC_CONDA_BUILD)
```

Comment:
So, this was set up this way because of the R build which uses `-DADBC_CONDA_BUILD=${CONDA_BUILD}` because of how `Makevars` in R doesn't allow a bunch of different Make constructs. 

So changing this like this will result in this being built for the R driver manager even when not using CONDA_BUILD

Reply:
It would be nice to document that :) I'll change it back